Automattic/woocommerce-subscriptions-core
# Check if WooCommerce is older than 7.3 in `wcs_get_page_screen_id()`.

Fixes #358

## Description







We want `wcs_get_page_screen_id( 'shop_subscription' )` to return 'shop_subscription' and since `wc_get_page_screen_id()` will do so since WC 7.3 (at least [it still returns an empty string in 7.2.2](https://github.com/woocommerce/woocommerce/blob/7.2.2/plugins/woocommerce/includes/admin/wc-admin-functions.php#L70-L76)), we should check `wcs_is_woocommerce_pre( '7.3.0' )` instead of `6.9.0`.

ps: I don't include changelog because this bug has not been released yet to public.

## How to test this PR





1. Install WC < 7.2.2.
2. Create a new product (/wp-admin/post-new.php?post_type=product).
3. You should not see a fatal error.

## Product impact
